libcudacxx/include/cuda/std/detail/libcxx/include/version

```diff
@@ -219,6 +219,11 @@ __cpp_lib_void_t                                        201411L <type_traits>
 #pragma GCC system_header
 #endif
 
+// We define certain feature test macros depending on availability
+#if NV_PROVIDES_SM_90
+# define __cccl_lib_has_experimental_ctk12_cp_async_exposure
```

Comment:
The `barrier::arrive_tx` method is not part of the experimental ctk12 cp async exposure. It is a planned non-experimental feature that we have prioritized, because it is very difficult to document `cp.async.bulk` without it.

I am not sure if it is common practice to add a feature flag for every new feature. It is is, then this flag should probably be renamed. If it is not common practice, then I propose we add this feature flag when the PTX wrappers for cp.async.bulk are merged.

Reply:
I would not say it is a common practice but definintely a good one. 

What would be a better name?

Reply:
I would go for  `__cccl_lib_has_local_barrier_arrive_tx`. There is also a "remote" barrier arrive. This will be implemented later. In this case, the feature is emulated on SM70 and SM80, so  I would also opt to make the feature test macro available on all architectures.